src/email/templating.rs

```diff
@@ -0,0 +1,19 @@
+use crate::http::structs::AybConfigEmail;
+
+const CLI_CONFIRM_TMPL: &str = "To complete your registration, type\n\tayb client confirm {token}";
+const WEB_CONFIRM_TMPL: &str = "To complete your registration, visit\n\t <{url}>";
```

Comment:
I feel silly, but what are the `<` and `>` for?

Reply:
Oh just muscle memory from Markdown syntax, I can remove it if you want

Reply:
I think it's good to remove it, since it will go in the email to a client that might not render Markdown

Reply:
Done! :)